product/base/scripts/ubuntu/install_quarto.sh

```diff
@@ -16,18 +16,34 @@ usage() {
     echo "  QUARTO_VERSION=1.3.340 $0"
     echo ""
     echo "Options:"
-    echo "  -d, --debug         Enable debug output"
-    echo "  -h, --help          Print usage and exit"
-    echo "      --prefix        Install Quarto to a custom prefix"
-    echo "                      Each version of Quarto will have its own subdirectory"
-    echo "                      Default: /opt/quarto"
+    echo "  -d, --debug                   Enable debug output"
+    echo "  -h, --help                    Print usage and exit"
+    echo "      --prefix                  Install Quarto to a custom prefix"
+    echo "                                Each version of Quarto will have its own subdirectory"
+    echo "                                Default: /opt/quarto"
+    echo "      --install-tinytex         Install TinyTeX using Quarto"
+    echo "      --add-path-tinytex        Add TinyTeX to PATH using Quarto"
+    echo "      --update-tinytex          Update TinyTeX using Quarto"
+    echo "      --uninstall-tinytex       Uninstall TinyTeX from Quarto"
 }
 
 
 # Set defaults
 PREFIX="/opt/quarto"
+QUARTO_PATH="${PREFIX}/${QUARTO_VERSION}/bin/quarto"
+ADD_PATH_TINYTEX=0
+INSTALL_TINYTEX=0
+UPDATE_TINYTEX=0
+UNINSTALL_TINYTEX=0
+IS_WORKBENCH_INSTALLATION=0
 
-OPTIONS=$(getopt -o hdr: --long help,debug,prefix: -- "$@")
+# Set Quarto Path to the bundled version in Workbench if it exists
+if [ -f "/lib/rstudio-server/bin/quarto/bin/quarto" ]; then
+    QUARTO_PATH="/lib/rstudio-server/bin/quarto/bin/quarto"
+    IS_WORKBENCH_INSTALLATION=1
+fi
+
+OPTIONS=$(getopt -o hdr: --long help,debug,prefix:,use-workbench-quarto,install-tinytex,add-path-tinytex,update-tinytex,uninstall-tinytex -- "$@")
```

Comment:
Can we remove parsing of the options that we aren't handling?
```suggestion
OPTIONS=$(getopt -o hd --long help,debug,prefix:,install-tinytex,add-path-tinytex,update-tinytex,uninstall-tinytex -- "$@")
```

Reply:
Good catch!